apps/experiments/models.py

```diff
@@ -1700,17 +1702,36 @@ def ad_hoc_bot_message(
         """
         try:
             with current_team(self.team):
-                bot_message = self._bot_prompt_for_user(instruction_prompt, trace_info, use_experiment=use_experiment)
-                self.try_send_message(message=bot_message)
+                experiment = use_experiment or self.experiment
+                trace_service = TracingService.create_for_experiment(experiment)
+                with trace_service.trace_or_span(
+                    name=f"{experiment.name} - {trace_info.name}",
+                    session_id=str(self.external_id),
+                    user_id=str(self.participant.identifier),
+                    inputs={"input": instruction_prompt},
+                    metadata=trace_info.metadata,
+                ):
+                    bot_message = self._bot_prompt_for_user(
+                        instruction_prompt, trace_info, use_experiment=use_experiment, trace_service=trace_service
+                    )
+                    self.try_send_message(message=bot_message)
+                    raise RuntimeError("Simulated bot failure!")
+                    trace_service.set_current_span_outputs({"response": bot_message})
+                    trace_info = trace_service.get_trace_metadata()
+                return trace_info
         except Exception as e:
             log.exception(f"Could not send message to experiment session {self.id}. Reason: {e}")
             if not fail_silently:
```

Comment:
I don't think there are any places that call this method with `fail_silently = True` so we can remove that param